arch/arc/core/thread.c

```diff
@@ -131,7 +131,7 @@ void _new_thread(struct k_thread *thread, k_thread_stack_t *stack,
 #endif
 	}
 
-	_new_thread_init(thread, pStackMem, stackSize, priority, options);
+	new_thread_init(thread, pStackMem, stackSize, priority, options);
```

Comment:
z_new_thread_init

Reply:
in kernel/device.c we have:
```
-static ALWAYS_INLINE void _new_thread_init(struct k_thread *thread,
+static ALWAYS_INLINE void new_thread_init(struct k_thread *thread,
```

Reply:
@pfl  ? This is defined only kernel/include/kernel_structs.h. I agree with @andrewboie, this should have zephry's namespace.

Reply:
_new_thread_init() is only defined in ./kernel/include/kernel_structs.h and there it is defined 'static ALWAYS_INLINE'. According to the initial understanding, static functions shall only have their initial underscores removed.
You are now saying that even though this (and other, there are more similar cases) function is static, it is anyway treated as a "global" and shall therefore start with 'z_' ?
Is this special handling only applicable to ./kernel/include/kernel_structs.h or what is thinking here?

Reply:
Fixed.